morceus-data/latin/stems/nominals/ls.nom

```diff
@@ -215264,15 +215404,17 @@ va^ti^a
 va_^trax		
 :le:vatrax
 :aj:va_^tr	ax_adj
+:aj:va_^tri^co_s	us_a_um masc variant
 
-
-va_^tri^co_sus		<itype>i</itype>
 Va_^trenus		
 :le:Vatrenus
 :no:Va_^tren	us_i masc
 
 
-ve^#1	
+ve^#1
+:le:ve
+:wd:ve^	conj
```

Comment:
is this the enclitic? Can we add an `enclitic` label here so it's clear?

Reply:
added enclitic label